### PR TITLE
DlgPrefRecord: fix initialization of default settings path

### PR DESCRIPTION
**Pull request by Be-ing**
Fixing https://bugs.launchpad.net/mixxx/+bug/1909982

### CODE REVIEW

File: src/preferences/dialog/dlgprefrecord.cpp [hunk 1]
```diff
@@ -383,13 +384,32 @@ void DlgPrefRecord::loadMetaData() {
 
 void DlgPrefRecord::saveRecordingFolder()
 {
```
Comment (ronso0) [inline src/preferences/dialog/dlgprefrecord.cpp:386]:
```suggestion
void DlgPrefRecord::saveRecordingFolder() {
```

Comment (ronso0) [reply src/preferences/dialog/dlgprefrecord.cpp:386]: puuh, it's like this in some places in this file..

Comment (Be-ing) [reply src/preferences/dialog/dlgprefrecord.cpp:386]: This is a bug fix PR. I am not doing a mass reformatting.

**Comment by Be-ing** [comment]
I don't know why this bug was only reported recently. The code has not been modified in 7 years.

**Comment by Holzhaus** [comment]
> The code has not been modified in 7 years.

Yeah, 2.3 has been in development for a looong time :smile:

File: src/preferences/dialog/dlgprefrecord.cpp (hunk 1 above)
Comment (ronso0) [reply src/preferences/dialog/dlgprefrecord.cpp:386]: sure.
I was under the impression we fix small style issues when touching a file, like QString > QStringLiteral.
604e317502
pick it if you like, or leave it

Comment (ronso0) [reply src/preferences/dialog/dlgprefrecord.cpp:386]: rest LGTM and works as expected

Comment (Be-ing) [reply src/preferences/dialog/dlgprefrecord.cpp:386]: I just don't even want to talk about formatting and consider such discussions a waste of time.

**Comment by ronso0** [comment]
I don't know why the github website sometimes selects "Sqash and merge" without me noticing.. sry

**Pull request merged**